packages/discord.js/src/managers/CachedManager.js

```diff
@@ -11,6 +11,13 @@ class CachedManager extends DataManager {
   constructor(client, holds, iterable) {
     super(client, holds);
 
+    /**
+     * The private cache of items for this manager.
```

Comment:
```suggestion
     * The private cache of items for this manager
```

Reply:
why without `.`

Reply:
It's hard to tell the correct format IMO. One property has a `.` in description, one doesn't 😅